app-frontend/src/app/pages/projects/edit/colormode/colormode.html

```diff
@@ -32,11 +32,9 @@ <h5 class="sidebar-title">Color Mode</h5>
         <ul class="sidebar-list">
           <li>
             <span class="label">Band</span>
-            <div class="dropdown btn-group fixedwidth" uib-dropdown uib-dropdown-toggle>
-              <a class="btn dropdown-label">
+            <div class="dropdown fixedwidth" uib-dropdown uib-dropdown-toggle>
```

Comment:
Not at this line in particular (and you had nothing to do with it), but we have a lot of options in the dropdown here. We could use a `max-height` or something on the `.dropdown-menu` class. It ends up adding scrollbars to the sidebar when I open the dropdown which is pretty jarring.

The issue is that the gradient dropdown has some unique scrolling as well (to keep the controls from scrolling out of view) so it doesn't seem like a one line fix or anything.

Not sure if you want to address this while you're in here or not.

Reply:
Yeah very true. I agree that the those dropdown menus need to be handled differently to address the large list. @designmatty and I were talking about transitioning those to `<select>` elements, which are more appropriate for form fields. Dropdowns are typically used for navigation, not for input.

I think we should keep this as is for now but consider whether we want to switch to `<select>` elements or try to fix the dropdowns for a future PR.

Reply:
:+1: